mysql-test/suite/rpl/t/rpl_column_compression_async_rpl.test

```diff
@@ -0,0 +1,32 @@
+#
+# PS-7806 : Column compression breaks async replication on PS
+#
+
+--source include/not_group_replication_plugin.inc
+--source include/master-slave.inc
```

Comment:
Also, please source `include/have_binlog_format_row.inc` as there is no need to run this test in all formats.